reference/test/matrix/coo_kernels.cpp

```diff
@@ -116,4 +105,127 @@ TEST_F(Coo, MovesToCsr)
 }
 
 
+TEST_F(Coo, ConvertsToDense)
```

Comment:
Here we first test the conversions, then the apply functions. In CSR we do it the other way round. I think this is not critical, but maybe we want to have the same order in all tests at some point.